### ISSUE TITLE
No suggestion after editing a variable.

### ISSUE THREAD

**Issue by mattez** [mrmlnc/vscode-less#17]
There is no suggestion after editing a variable.

Example:
I want variable ```@font-size-h1``` from BS3.
So I wrote ```@fsh1``` and suggestion works, so I've got ```@font-size-h1```
![vsc-less-01](https://cloud.githubusercontent.com/assets/143115/19625678/71bf69ba-991f-11e6-9321-a1f4af67ab89.png)
Than I delete some characters from variable, cursor on variable and use
```json
"key": "ctrl+space",            "command": "editor.action.triggerSuggest"
```
![vsc-less-02](https://cloud.githubusercontent.com/assets/143115/19625687/b64cf05c-991f-11e6-8325-a3421fc58d62.png)
no suggestion there.

Is this issue of "vscode-less"? It will be nice if it can suggest also after deleting some part oaf variable.

**Comment by mrmlnc** [comment]
Will be fixed in version 0.3.0, which will be released today.

**Comment by mattez** [comment]
You are unbelievable. Super fast. Mind control. :+1:   
Have a nice Sunday.
